conf/init-lsp.el

```diff
@@ -35,7 +35,9 @@
   (lsp-keymap-prefix nil)
   (lsp-session-file (me/cache-concat "lsp/session.eld")))
 
-(use-package lsp-ui
```

Comment:
You want to leave it here and just say

```elisp
(use-package lsp-ui
  :disabled t
  :commands lsp-ui-mode)
```

I think that's more common in emacs conf

Reply:
Yeah, that's infinitely less dumb. Fixed.